samples/nrf9160/nrf_cloud_rest_fota/README.rst

```diff
@@ -0,0 +1,99 @@
+.. _nrf_cloud_rest_fota:
+
+nRF9160: nRF Cloud REST FOTA
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The REST FOTA sample demonstrates how to use the `nRF Cloud REST API`_ to perform FOTA updates on your device.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf9160dk_nrf9160_ns
+
+The sample requires an `nRF Cloud`_ account.
+
+.. note::
+   This sample requires modem firmware v1.3.x or later.
+
+Overview
+********
+
+Firmware updates can be managed on the `nRF Cloud`_ website or by using the `nRF Cloud REST API`_ directly.
+See the `nRF Cloud Getting Started FOTA documentation`_ for details.
+
+User interface
+**************
+
+Pressing the button configured by :kconfig:`CONFIG_REST_FOTA_BUTTON_EVT_NUM` will bypass the wait time specified by :kconfig:`CONFIG_REST_FOTA_JOB_CHECK_RATE_MIN` and immediately perfom an update check.
+
+If :kconfig:`CONFIG_REST_FOTA_DO_JITP` is enabled, pressing the button configured by :kconfig:`CONFIG_REST_FOTA_BUTTON_EVT_NUM`, when prompted at startup, will request just-in-time provisioning via REST with nRF Cloud.
+This is useful when initially provisioning and associating a device on nRF Cloud.
+You only need to do this once per device.
+
+On startup, if JITP has not been requested the sample will ask if FOTA should be enabled in the device's shadow. Press the button configured by :kconfig:`CONFIG_REST_FOTA_BUTTON_EVT_NUM` to perform the shadow update.
+Depending on how you provisioned your device with nRF Cloud, this step might not be needed. You only need to do this once per device.
+
+If :kconfig:`CONFIG_REST_FOTA_ENABLE_LED` is enabled, an LED configured by :kconfig:`CONFIG_REST_FOTA_LED_NUM` indicates the state of the connection to the LTE network.
+
+Configuration
+*************
+|config|
+
+Configuration options
+=====================
+
+Check and configure the following configuration options for the sample:
+
+.. option:: CONFIG_REST_FOTA_JOB_CHECK_RATE_MIN - Update check rate
+
+This configuration option defines how often the sample will check for FOTA updates.
```

Comment:
```suggestion
.. kconfig:: CONFIG_REST_FOTA_JOB_CHECK_RATE_MIN

CONFIG_REST_FOTA_JOB_CHECK_RATE_MIN - Update check rate
   This configuration option defines how often the sample checks for FOTA updates.
```
Format all sample-specific Kconfig options like this. This creates a link from the option mentioned earlier in this doc to this description.